judge/tasks/submission.py

```diff
@@ -18,11 +18,40 @@ def apply_submission_filter(queryset, id_range, languages, results):
         queryset = queryset.filter(language_id__in=languages)
     if results:
         queryset = queryset.filter(result__in=results)
-    queryset = queryset.exclude(locked_after__lt=timezone.now()) \
+    queryset = queryset.exclude(is_archived=True) \
+                       .exclude(locked_after__lt=timezone.now()) \
```

Comment:
Why is this added? I dont see it immediately, what is this function for?

Reply:
this queryset is used by `/problem/aplusb/manage/submission > Rejudge selected submissions`

`.exclude(is_archived=True)` is not needed anymore, and i'll remove it